HelpSource/Classes/Quark.schelp

```diff
@@ -110,6 +110,10 @@ returns:: this
 METHOD:: version
 returns:: String
 
+METHOD:: branch
+returns the current branch name.
+returns:: String or nil, if failed
```

Comment:
not something we can address in this PR, but one of the complaints we get about Quarks is that it doesn't do a good job of notifying about failure. i think what you've done here is consistent with the rest of the API but the whole thing is probably worth revisiting at some point

Reply:
yes, true. It is a bit a can of worms, but maybe we find a simple way.